minecraft/1.7/src/main/java/nova/core/wrapper/mc/forge/v17/wrapper/item/ItemWrapperMethods.java

```diff
@@ -89,19 +94,29 @@ default boolean shouldUseRenderHelper(IItemRenderer.ItemRenderType type, ItemSta
 	@Override
 	default void renderItem(IItemRenderer.ItemRenderType type, ItemStack itemStack, Object... data) {
 		Item item = Game.natives().toNova(itemStack);
-		BWModel model = new BWModel();
-
-		if (item.components.has(StaticRenderer.class)) {
-			StaticRenderer staticRenderer = item.components.get(StaticRenderer.class);
-			staticRenderer.onRender.accept(model);
-		} else if (item.components.has(DynamicRenderer.class)) {
-			DynamicRenderer dynamicRenderer = item.components.get(DynamicRenderer.class);
-			dynamicRenderer.onRender.accept(model);
+		if (item.components.has(Renderer.class)) {
+			GL11.glPushAttrib(GL_TEXTURE_BIT);
+			GL11.glEnable(GL12.GL_RESCALE_NORMAL);
+			GL11.glPushMatrix();
+			Tessellator.instance.startDrawingQuads();
+			BWModel model = new BWModel();
+			model.matrix.rotate(Direction.UP.toVector(), 1 / 4 * Math.PI);
+			model.matrix.rotate(Direction.EAST.toVector(), 1 / 6 * Math.PI);
+			model.matrix.scale(1.6, 1.6, 1.6);
+
+			if (item.components.has(StaticRenderer.class)) {
```

Comment:
Use `if (item.components.has(Renderer.class))` and `item.components.getSet(Renderer.class).forEach(...)` here.